src/modules/launcher/Plugins/Microsoft.Plugin.Registry/Properties/Resources.resx

```diff
@@ -0,0 +1,174 @@
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="PluginTitle" xml:space="preserve">
+    <value>Registry Plugin</value>
+  </data>
+  <data name="PluginDescription" xml:space="preserve">
+    <value>Search inside the Windows Registry</value>
+  </data>
+  <data name="CopyKeyPath" xml:space="preserve">
+    <value>Copy key path</value>
+  </data>
+  <data name="Key" xml:space="preserve">
+    <value>Key:</value>
+  </data>
+  <data name="Name" xml:space="preserve">
+    <value>Name:</value>
+  </data>
+  <data name="CopyValueName" xml:space="preserve">
+    <value>Copy value name</value>
+  </data>
+  <data name="OpenKeyInRegistryEditor" xml:space="preserve">
+    <value>Open key in registry editor</value>
+  </data>
+  <data name="OpenInRegistryEditorAccessExceptionText" xml:space="preserve">
+    <value>You do not have enough rights to open the Windows registry editor</value>
+  </data>
+  <data name="OpenInRegistryEditorAccessExceptionTitle" xml:space="preserve">
+    <value>Error on open Registry Editor</value>
+  </data>
+  <data name="SubKeys" xml:space="preserve">
+    <value>Sub-keys:</value>
+  </data>
+  <data name="Values" xml:space="preserve">
+    <value>Values:</value>
+  </data>
+  <data name="RegistryKey" xml:space="preserve">
+    <value>Registry key</value>
+  </data>
+  <data name="RegistryValue" xml:space="preserve">
+    <value>Registry Value</value>
+  </data>
+  <data name="Type" xml:space="preserve">
+    <value>Type:</value>
```

Comment:
Comment: See https://docs.microsoft.com/en-us/windows/win32/sysinfo/registry-value-types for proper context of how to translate 'type'